article-analyzer/src/main/java/co/inventorsoft/academy/spring/config/ApplicationConfig.java

```diff
@@ -0,0 +1,24 @@
+package co.inventorsoft.academy.spring.config;
+
+import com.google.gson.Gson;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.List;
+
+@Configuration
+public class ApplicationConfig {
+    @Value("${excluded.words}")
+    private List<String> excludedWords;
+
+    @Bean
+    public List<String> excludedWords() {
+        return excludedWords;
+    }
```

Comment:
that is a cool idea of how u can create a bin of a collection, a bit too much, but let it be